x-pack/platform/plugins/private/reporting/server/routes/common/request_handler/schedule_request_handler.ts

```diff
@@ -15,7 +15,7 @@ import { SavedObjectsUtils } from '@kbn/core/server';
 import type { IKibanaResponse } from '@kbn/core/server';
 import { ScheduleType } from '@kbn/reporting-server';
 import type { RawNotification } from '../../../saved_objects/scheduled_report/schemas/latest';
-import { rawNotificationSchema } from '../../../saved_objects/scheduled_report/schemas/v1';
+import { rawNotificationSchema } from '../../../saved_objects/scheduled_report/schemas/v4';
```

Comment:
nit: could probably change this to use `schemas/latest`

Reply:
I wanted to, but I saw that in `/latest` we only export types, not the entire schemas. I guess adding the schemas too wouldn't hurt?

Reply:
i imagine it wouldn't hurt. probably an oversight (by me!)